includes/admin/class-wc-admin-menus.php

```diff
@@ -346,7 +346,7 @@ public function admin_bar_menus( $wp_admin_bar ) {
 		}
 
 		// Show only when the user is a member of this site, or they're a super admin.
-		if ( ! is_user_member_of_blog() && ! is_super_admin() ) {
+		if ( ! is_super_admin() ) {
```

Comment:
A user that has access to the dashboard on a site but is not super admin on the network will stop seeing the "Visit store" link after this change. Was this change intentional?

Reply:
I can confirm this. I don't think the original logic needed changing in this case.

Reply:
Shop managers are mostly who it would affect